webapp/src/components/MessageList.tsx

```diff
@@ -20,19 +24,25 @@ const MessageList: FC<MessageListProps> = ({
   projectName,
   translations,
 }) => {
+  const theme = useTheme();
   const [height, setHeight] = useState<number | undefined>(undefined);
+  const [layout, setLayout] = useState<MessageFormLayout>('linear');
+
+  const onResize = useCallback(() => {
+    setHeight(window.innerHeight);
+    setLayout(
+      window.innerWidth > theme.breakpoints.values.lg ? 'grid' : 'linear',
```

Comment:
Can we `useMediaQuery` instead of a `resize` handler and state?
https://mui.com/material-ui/react-use-media-query/#using-material-uis-breakpoint-helpers

Reply:
Nice catch. Fixed in https://github.com/zetkin/lyra/pull/120/commits/58571bfd11c197181c67520524c0df81fb896753. We're stuck with the setHeight(window.innerHeight) though. Nothing in MUI for that.